### DISCUSSION TITLE
Restrict remove rights

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#81] [Idea retired / denied]
Currently, you can simply remove all rights in a tenant. Under certain circumstances, this leads to the fact that, for example, no one has access to the access management anymore and therefore nothing can be adjusted.

How about if there must always be one person with the right "access management"? Or that the "Tenant owner/creator" cannot remove this right for himself?

**Comment by YvesR** [comment]
IMHO it is not worth it to implement such a security mechanism. For this rare situation we still could adjust permissions in a support case done by DevOps team.
